pwa/app/routes/about.tsx

```diff
@@ -0,0 +1,145 @@
+import { Button } from '~/components/ui/button';
+
+export default function About(): JSX.Element {
+  return (
+    <>
+      <div className="flex flex-col divide-y [&_*]:mt-4">
+        <div>
+          <h1 className="text-3xl font-medium">About Us</h1>
+          <p>
+            Founded in the fall of 2006, The Blue Alliance began as a website
+            dedicated to providing everyone involved in the FIRST Robotics
+            Competition (FRC) with scouting data and match videos. Since then,
+            the project has grown to include developers from within the FIRST
+            community from various teams all over the world. We continually
+            strive to make The Blue Alliance an even more valuable resource for
+            our users. Other additions to The Blue Alliance include match
+            prediction and analytics, hosting of CAD models, and The Blue
+            Alliance Blog.
+          </p>
+          <p>
+            Since 2022, The Blue Alliance has been a 501(c)(3) non-profit
+            organization. We rely on community donations to pay for hosting
+            costs.
+          </p>
+
+          <p>
+            You can support The Blue Alliance or reach us with the following:
+          </p>
+          <Button
+            onClick={() =>
```

Comment:
Shouldn't manually be setting the window location. External sites should just be a normal `<a>` that's formatted to look like a button.